README.md

```diff
@@ -1,6 +1,97 @@
-# PatternFlow
```

Comment:
Commits of the algorithm have been done in the main algorithms directory. Will overwrite the algorithms Readme file. Please move you algorithm into its own directory under the correct sub directory (transform in this case)